tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/to/AspectToSchemaCommand.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package io.openmanufacturing.sds.aspect.to;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.openmanufacturing.sds.AbstractCommand;
+import io.openmanufacturing.sds.aspect.AspectToCommand;
+import io.openmanufacturing.sds.aspectmodel.generator.jsonschema.AspectModelJsonSchemaGenerator;
+import io.openmanufacturing.sds.exeption.CommandException;
+import io.openmanufacturing.sds.metamodel.Aspect;
+import io.openmanufacturing.sds.metamodel.loader.AspectModelLoader;
+import picocli.CommandLine;
+
+@CommandLine.Command( name = AspectToSchemaCommand.COMMAND_NAME,
```

Comment:
Maybe this should be renamed to `AspectToJsonSchemaCommand` since in theory we could also write other "schema"s, such das RDF Schema or XML Schema.